tests/test_flatten.py

```diff
@@ -31,3 +47,55 @@ def test_flatten_dict():
             "contextInfo_context2": "info2",
         }
     ]
+
+
+def test_flatten_pd():
+    flattened_pd = flatten(
+        PD_DATA, nested_columns=["contextInfo", "neighborhood"]
+    )
+    assert flattened_pd.equals(
+        DataFrame(
+            [
+                {
+                    "answer": 42,
+                    "contextInfo_context1": "info1",
+                    "contextInfo_context2": "info2",
+                }
+            ]
+        )
+    )
+
+
+def test_flatten_df_is_called():
+    mock_flatten_df = Mock()
+
+    with patch("crossfire._flatten_df", mock_flatten_df):
```

Comment:
Please, review the docs for `patch` (or other use cases in this project)  because this test does not do what you think it is doing. It mocks, but the mock you assert is not the patched one (it is the one you created manually on line 70).